nannou/src/window.rs

```diff
@@ -726,9 +726,10 @@ impl<'app> Builder<'app> {
         self
     }
 
+    #[cfg(not(target_os = "unknown"))]
     /// Builds the window, inserts it into the `App`'s display map and returns the unique ID.
     pub fn build(self) -> Result<Id, BuildError> {
-        futures::executor::block_on(self.build_async())
+        async_std::task::block_on(self.build_async())
     }
 
     pub async fn build_async(self) -> Result<Id, BuildError> {
```

Comment:
We could think about putting `#[cfg(target_os = "unknown")]` or `#[cfg(feature = "wasm-experimental")]` here s.t. the `async` we are flexible to change the `async` concept without introducing breaking changes to "normal" users i.e. those that do not use `wasm-experimental`.